src/plugins/workspace/public/components/workspace_overview_app.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect } from 'react';
+import { I18nProvider } from '@osd/i18n/react';
+import { CoreStart } from 'opensearch-dashboards/public';
+import { useObservable } from 'react-use';
+import { EuiBreadcrumb } from '@elastic/eui';
+import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';
+import { WorkspaceOverview, WorkspaceOverviewProps } from './workspace_overview/workspace_overview';
+
+export const WorkspaceOverviewApp = (props: WorkspaceOverviewProps) => {
+  const {
+    services: { workspaces, chrome, application },
+  } = useOpenSearchDashboards<CoreStart>();
+
+  const currentWorkspace = useObservable(workspaces.currentWorkspace$);
+
+  /**
+   * set breadcrumbs to chrome
+   */
+  useEffect(() => {
+    const breadCrumbs: EuiBreadcrumb[] = [
```

Comment:
I see some places are using `breadCrumbs` and some are using `breadcrumbs`, shall we align them?

Reply:
Thanks, changed to breadcrumbs